src/pages/inventory/inventory.ts

```diff
@@ -22,9 +22,11 @@ export class InventoryPage {
   selectedModelID = -1;
   allCategories;
   selectedCategoryID = -1;
-  allItems;
-  filteredItems;
+  items;
   showFilters = false;
+  limit = 10;
```

Comment:
Magic number alert 🚨

I think it would make sense to a) move it to a constants file, b) rename it `defaultLimit`, or c) add a comment explaining why the value is `10`.